aws/resource_aws_kms_grant_test.go

```diff
@@ -168,6 +170,26 @@ func TestAccAWSKmsGrant_ARN(t *testing.T) {
 	})
 }
 
+func TestAccAWSKmsGrant_disappers(t *testing.T) {
+	timestamp := time.Now().Format(time.RFC1123)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSKmsGrantDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSKmsGrant_Basic("basic", timestamp, "\"Encrypt\", \"Decrypt\""),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSKmsGrantExists("aws_kms_grant.basic"),
+					testAccCheckAWSKmsGrantDisappears("aws_kms_grant.basic"),
+				),
+				ExpectNonEmptyPlan: true,
+			},
+		},
+	})
```

Comment:
Due to the merge of another pull request, the input parameters for `testAccAWSKmsGrant_Basic` changed to remove the timestamp. Can you please update to the following?

```suggestion
	resourceName := "aws_kms_grant.test"
	rName := acctest.RandomWithPrefix("tf-acc-test")

	resource.ParallelTest(t, resource.TestCase{
		PreCheck:     func() { testAccPreCheck(t) },
		Providers:    testAccProviders,
		CheckDestroy: testAccCheckAWSKmsGrantDestroy,
		Steps: []resource.TestStep{
			{
				Config: testAccAWSKmsGrant_Basic(rName, "\"Encrypt\", \"Decrypt\""),
				Check: resource.ComposeTestCheckFunc(
					testAccCheckAWSKmsGrantExists(resourceName),
					testAccCheckAWSKmsGrantDisappears(resourceName),
				),
				ExpectNonEmptyPlan: true,
			},
		},
	})
```

Reply:
Updated. Thanks for your code suggestion. I haven't used this nice GitHub "Add suggest to batch" and "Commit suggestion" features before. Pretty convenient.